client/components/Problem/Problem.scss

```diff
@@ -0,0 +1,101 @@
+.problem {
+  overflow: hidden;
+  margin-bottom: 100px;
+  width: 625px;
+  font-family: Slabo, sans-serif;
+}
+
+.rating {
+  float: left;
+  padding: 5px 0 0 5px;
+  border-radius: 5px;
+  height: 35px;
+  width: 50px;
+  background-color: white;
+  font-size: 25px;
+  color: red;
+}
+
+.user-name {
+  float: left;
+  margin-top: 6px;
+  padding: 5px 0 0 5px;
+  height: 25px;
+  width: 400px;
+  background: rgba(200, 200, 200, 1);
+  font-size: 16px;
+}
+
+.subject {
+  float: left;
+  margin-top: 3px;
+  padding: 5px 15px 5px 15px;
+  border-radius: 5px;
+  width: auto;
+  background: rgba(77, 0, 0, 1);
+  font-size: 20px;
+  color: white;
+
+  i {
```

Comment:
As I said before - if you need style something - add class on that node. Avoid css selectors nesting